modules/ROOT/pages/troubleshooting-tips.adoc

```diff
@@ -94,68 +113,91 @@ to the client to force a downgrade: PROTOCOL (code=Code Protocol [0x0000000A],
+
+You can also pass the `-version` flag to the {product-proxy} to print the version.
+For example, you can use the following Docker command:
 
 [source,bash]
 ----
 docker run --rm datastax/zdm-proxy:2.x -version
 ZDM proxy version 2.1.0
 ----
 
-[TIP]
+[IMPORTANT]
 ====
-Do not use `--rm` when actually launching the {product-proxy} otherwise you will not be able to access the logs when it stops (or crashes).
+Don't use `--rm` when you launch the {product-proxy} container.
+This flag will prevent you from accessing the logs when {product-proxy} stops or crashes.
 ====
 
-[#how-to-leverage-metrics]
-== How to leverage the metrics provided by {product-proxy}
+== Report an issue
+
+To report an issue or get additional support, submit an issue in the {product-short} component GitHub repositories:
 
-See xref:metrics.adoc[].
+* https://github.com/datastax/zdm-proxy/issues[{product-proxy} repository]
+* https://github.com/datastax/zdm-proxy-automation/issues[{product-automation} repository] (includes {product-automation} and the {product-utility})
+* https://github.com/datastax/cassandra-data-migrator/issues[{cass-migrator} repository]
+* https://github.com/datastax/dsbulk-migrator/issues[{dsbulk-migrator} repository]
 
-== Reporting an issue
+[IMPORTANT]
+====
+These repositories are public.
+
+Don't include any proprietary or private information in issues, pull requests, or comments that you make in these repositories.
+====
 
-If you encounter a problem during your migration, please contact us.
-In the {product-proxy} GitHub repo, submit a https://github.com/datastax/zdm-proxy/issues[GitHub Issue].
-Only to the extent that the issue's description does not contain **your proprietary or private** information, please include the following:
+If possible, include the following information in the issue description, and remove all proprietary and private information before submitting the issue:
 
-* {product-proxy} version (see xref:_how_to_identify_the_zdm_proxy_version[here]).
-* {product-proxy} logs: ideally at `debug` level if you can reproduce the issue easily and can tolerate a restart of the proxy instances to apply the configuration change.
-* Version of database software on the Origin and Target clusters (relevant for {dse-short} and {cass} deployments only).
-* If {astra-db} is being used, please let us know in the issue description.
-* Screenshots of the {product-proxy} metrics dashboards from Grafana or whatever visualization tool you use.
-If you can provide a way for us to access those metrics directly that would be even better.
-* Application/Driver logs.
-* Driver and version that the client application is using.
+* Your <<check-version,{product-proxy} version>>.
 
-=== Reporting a performance issue
+* <<proxy-logs,{product-proxy} logs>>, ideally at `debug` level, if you can easily reproduce the issue and tolerate restarting the proxy instances to apply the log level configuration change.
```

Comment:
Sorry if I'm making a mess of the log leveling. I'm assuming you're using `CAPS` to refer to the log level configuration setting, and `smalls` to refer to individual messages/entries.

```suggestion
* <<proxy-logs,{product-proxy} logs>>, ideally at `DEBUG` level, if you can easily reproduce the issue and tolerate restarting the proxy instances to apply the log level configuration change.
```